concent_api/concent_api/tests/test_check_middleman_port.py

```diff
@@ -0,0 +1,69 @@
+from django.test import override_settings
+from django.conf import settings
+from django.test import TestCase
+
+from concent_api.system_check import check_middleman_port
+from concent_api.system_check import create_error_45_middleman_port_is_not_set
+from concent_api.system_check import create_error_46_middleman_port_has_wrong_type
+from concent_api.system_check import create_error_47_middleman_port_has_wrong_value
+
+
+class TestMiddlemanPortCheck(TestCase):
+
+    @override_settings(
+        CONCENT_FEATURES=[
+            'middleman'
+        ]
+    )
+    def test_that_middleman_port_not_set_will_produce_error_when_middleman_is_in_concent_features(self):
+        del settings.MIDDLEMAN_PORT
+
+        errors = check_middleman_port()
+
+        self.assertEqual(len(errors), 1)
+        self.assertEqual(errors[0], create_error_45_middleman_port_is_not_set())
+
+    @override_settings(
+        CONCENT_FEATURES=[]
+    )
```

Comment:
I'm wondering which of this styles would be better:
```python
@override_settings(CONCENT_FEATURES=[])
```
or
```python
@override_settings(
   CONCENT_FEATURES=[]
)
```
Because I think that breaking one line statement to another line is not necessary.

Reply:
I always do this for readability. I like when code is not extended to much to the right, and I can read it from top to bottom, but I understand someone can have different feelings about it. Actually this specific breaking for `@override_settings` is already used in other places do I though it would be ok. But I think if we want to do make it clear it should be explicitly described in our coding style.

Reply:
I started to think about this when @cameel said that we sometimes breaks lines too much. That's why I started this discuss :)

Reply:
Both versions are fine and allowed in our style.

@rwrzesien's preferred version has the advantage that you don't have to rewrap it when the line gets longer. But it makes the code less compact.

I personally prefer `@override_settings(CONCENT_FEATURES=[])` when the line fits below the suggested length limit (120 characters) but it's not a rule, just my preference.

Reply:
> But it makes the code less compact.

I think this actually increases readability.

Reply:
Not always. When the code is complex, it does. But when it's simple it may actually make it look more complex than it really is.

This is pretty compact:
``` python
@override_settings(CONCENT_FEATURES=[])
@override_settings(BLENDER_THREADS=5)
@override_settings(VERIFIER_DOWNLOAD_CHUNK_SIZE=1000)
@override_settings(VERIFIER_STORAGE_PATH='/var/verifier/')
@override_settings(STATICFILES_DIRS=['/a/b/c'])
@override_settings(AUTH_PASSWORD_VALIDATORS=[{'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator'}])
@override_settings(CUSTOM_PROTOCOL_TIMES=True)
def test(self):
    pass
```

and for me adding more whitespace does not make it more readable:
``` python
@override_settings(
    CONCENT_FEATURES=[]
)
@override_settings(
    BLENDER_THREADS=5
)
@override_settings(
    VERIFIER_DOWNLOAD_CHUNK_SIZE=1000
)
@override_settings(
    VERIFIER_STORAGE_PATH='/var/verifier/'
)
@override_settings(
    STATICFILES_DIRS=[
        '/a/b/c'
    ]
)
@override_settings(
    AUTH_PASSWORD_VALIDATORS=[
        {
            'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator'
        }
    ]
)
@override_settings(
    CUSTOM_PROTOCOL_TIMES=True
)
def test(self):
    pass
```